### PR TITLE
Separate headfile operators.hpp into multiple files.

### PR DESCRIPTION
**Pull request by scxiao**
This is for the issue #181.

### CODE REVIEW

File: src/eliminate_contiguous.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include <migraphx/eliminate_contiguous.hpp>
 #include <migraphx/program.hpp>
 #include <migraphx/instruction.hpp>
-#include <migraphx/operators.hpp>
+//#include <migraphx/op/operators.hpp>
```
Comment (kahmed10) [inline src/eliminate_contiguous.cpp]: This should be removed

File: src/eliminate_identity.cpp [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #include <migraphx/eliminate_identity.hpp>
 #include <migraphx/program.hpp>
 #include <migraphx/instruction.hpp>
-#include <migraphx/operators.hpp>
+//#include <migraphx/op/operators.hpp>
```
Comment (kahmed10) [inline src/eliminate_identity.cpp]: this should be removed

File: src/include/migraphx/operators.hpp [hunk 3]
```diff
@@ -1,1402 +0,0 @@
...
-        os << x.name();
-        return os;
-    }
-};
-
-} // namespace op
-} // namespace MIGRAPHX_INLINE_NS
-} // namespace migraphx
-
-#endif
```
Comment (pfultz2) [inline src/include/migraphx/operators.hpp]: We should not remove this header. We can have this header include every header under op directory.

File: src/include/migraphx/concat_opt.hpp [hunk 4]
```diff
@@ -9,7 +9,7 @@
 #include <utility>
 
 #include <migraphx/operation.hpp>
-#include <migraphx/operators.hpp>
+#include <migraphx/op/concat.hpp>
```
Comment (pfultz2) [inline src/include/migraphx/concat_opt.hpp]: This change needs to be update in the `concat_opt.hpp` file under the `tools/include`. And then you need to regenerate these files with `make generate`.

File: src/include/migraphx/operators.hpp (hunk 3 above)
Comment (scxiao) [reply src/include/migraphx/operators.hpp]: fixed

File: src/include/migraphx/concat_opt.hpp (hunk 4 above)
Comment (scxiao) [reply src/include/migraphx/concat_opt.hpp]: fixed

File: src/eliminate_contiguous.cpp (hunk 1 above)
Comment (scxiao) [reply src/eliminate_contiguous.cpp]: fixed

**Pull request merged**